Marlin/Marlin_main.cpp

```diff
@@ -4829,6 +4844,66 @@ inline void gcode_M31() {
 
 #endif // SDSUPPORT
 
+#if ENABLED(PISTON_EXTRUDER)
+  inline void gcode_M35() { //turn on dryer fan
```

Comment:
Looking at the CNC M6/M7, it makes sense for the fan on/off (now M35/M36).
The extruder rate adjustment, M37 is complex and has nothing to do with coolant, M8 - I could use it instead of M37 even though it would cause re-write of some other stuff.
Not a serious problem for me,.
Should I make these and the above changes and submit another pull request?